gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/utils/GenericThreadLauncher.java

```diff
@@ -90,7 +90,8 @@ private boolean checkInterruption(Exception exception) {
         boolean isInterrupt = false;
```

Comment:
Can you rename it "isInterrupted" please ?

Reply:
ok will do